src/psyclone/configuration.py

```diff
@@ -97,8 +97,7 @@ class Config(object):
     _instance = None
 
     # List of supported API by PSyclone
-    _supported_api_list = ["dynamo0.1", "dynamo0.3",
-                           "gocean0.1", "gocean1.0", "nemo"]
+    _supported_api_list = ["dynamo0.1", "dynamo0.3", "gocean1.0", "nemo"]
```

Comment:
Copyright date?

Reply:
Since we're here, would you mind removing the `u` prefix from the string at line 107 - pylint points out that it is redundant.

Reply:
Done